integrations/discord/events.go

```diff
@@ -13,15 +13,26 @@ import (
 func (h *handler) handleEvent(event any, eventType string) {
 	l := h.logger.With(zap.String("eventType", eventType))
 
-	// TODO(ENG-1546) - Add support for more event types
-	var authorID string
+	var initiatorID string
 	switch e := event.(type) {
 	case *discordgo.MessageCreate:
-		authorID = e.Author.ID
+		initiatorID = e.Author.ID
 	case *discordgo.MessageUpdate:
-		authorID = e.Author.ID
+		initiatorID = e.Author.ID
 	case *discordgo.MessageDelete:
-		authorID = "" // Deleted messages don't have an author
+		initiatorID = "" // Deleted messages don't have an author
+	case *discordgo.MessageReactionAdd:
+		initiatorID = e.UserID
+	case *discordgo.MessageReactionRemove:
+		initiatorID = e.UserID
+	case *discordgo.PresenceUpdate:
+		initiatorID = e.User.ID
+	case *discordgo.ThreadCreate:
+		initiatorID = e.OwnerID
+	case *discordgo.ThreadUpdate:
```

Comment:
Clarification question: what does the thread update even mean? Is it only the creator of the thread who can update it?

Reply:
In the case of update, I'm assuming the "owner" ID belongs to the original creator. Is there an ID of who initiated the current change, which may be different than the ID of the thread owner?

Reply:
There is not... Confirmed on the `discordgo` discord channel. 
chatGPT says to check the audit logs which sounds like a pain... Guy on `discordgo` confirmed this.